### ISSUE TITLE
5$: trailing comma in return should report error

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#913]
```javascript
export function datestampToRFC822(datestamp) {
  let match = datestamp.match(
    /^(?<year>\d{4})-(?<month>\d{2})-(?<day>\d{2})$/
  );
  if (!match) {
    throw new Error(`failed to parse timestamp: ${datestamp}`);
  }
  let { year, month, day } = match.groups;
  let monthNumber = parseInt(month, 10);
    // See: https://www.w3.org/Protocols/rfc822/#z28
    return `${day} ${
      rfc822Months[monthNumber - 1]
    } ${year} 00:00:00 Z`,
}
```
quick-lint-js reports no error on the second-to-last line, but it should.

**Comment by TomBinford** [comment]
Hi! I'd like to take this one.
